docker/Dockerfile.spark

```diff
@@ -0,0 +1,15 @@
+FROM apache/spark:v3.3.1
+
+USER root
+
+WORKDIR /opt/spark
+
+RUN apt-get update && apt-get install -y vim wget software-properties-common ssh net-tools ca-certificates python3 python3-pip
+
+RUN update-alternatives --install "/usr/bin/python" "python" "$(which python3)" 1
```

Comment:
This command seems more complicated than expected. Can we do something like `apt-get install python3.7`?